examples/letter_spacing.js

```diff
@@ -0,0 +1,137 @@
+
+import * as THREE from 'three';
+import { VRButton } from 'three/examples/jsm/webxr/VRButton.js';
+import { OrbitControls } from 'three/examples/jsm/controls/OrbitControls.js';
+import { BoxLineGeometry } from 'three/examples/jsm/geometries/BoxLineGeometry.js';
+
+import ThreeMeshUI from '../src/three-mesh-ui.js';
+
+import FontJSON from './assets/Roboto-msdf.json';
+import FontImage from './assets/Roboto-msdf.png';
+
+const WIDTH = window.innerWidth;
+const HEIGHT = window.innerHeight;
+
+let scene, camera, renderer, controls ;
+
+window.addEventListener('load', init );
+window.addEventListener('resize', onWindowResize );
+
+//
+
+function init() {
+
+	scene = new THREE.Scene();
+	scene.background = new THREE.Color( 0x505050 );
+
+	camera = new THREE.PerspectiveCamera( 60, WIDTH / HEIGHT, 0.1, 100 );
+
+	renderer = new THREE.WebGLRenderer({
+		antialias: true
+	});
+	renderer.setPixelRatio( window.devicePixelRatio );
+	renderer.setSize( WIDTH, HEIGHT );
+	renderer.xr.enabled = true;
+	document.body.appendChild(VRButton.createButton(renderer));
+	document.body.appendChild( renderer.domElement );
+
+	controls = new OrbitControls( camera, renderer.domElement );
+	camera.position.set( 0, 1.6, 0 );
+	controls.target = new THREE.Vector3( 0, 1, -1.8 );
+	controls.update();
+
+	// ROOM
+
+	const room = new THREE.LineSegments(
+		new BoxLineGeometry( 6, 6, 6, 10, 10, 10 ).translate( 0, 3, 0 ),
+		new THREE.LineBasicMaterial( { color: 0x808080 } )
+	);
+
+	scene.add( room );
+
+	// TEXT PANEL
+
+	makeTextPanel();
+
+	//
+
+	renderer.setAnimationLoop( loop );
+
+};
+
+//
+
+function makeTextPanel() {
+
+	const container = new ThreeMeshUI.Block({
+		width: 1.2,
+		height: 0.5,
+		padding: 0.05,
+		justifyContent: 'center',
+		alignContent: 'left',
+		fontFamily: FontJSON,
+		fontTexture: FontImage
+	});
+
+	container.position.set( 0, 1, -1.8 );
+	container.rotation.x = -0.55;
+	scene.add( container );
+
+	//
+
+	const text = new ThreeMeshUI.Text({
+			content: "This library supports line-break-friendly-characters, and letter spacing,",
+			fontSize: 0.055
+		});
+
+	container.add(
+
+		text,
+
+		new ThreeMeshUI.Text({
+			content: " As well as multi-font-size lines with consistent vertical spacing.",
+			fontSize: 0.08
+		})
+
+	);
```

Comment:
I think the example is useful, but you should make the text more explanatory. For example you could remove the second `Text` component and update the first one's content to `"letterSpacing ".repeat(3)`.